hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/ReadableNodeStoreImpl.java

```diff
@@ -68,7 +68,7 @@ public long sizeOfState() {
         return nodesState.size();
     }
 
-    protected <T extends ReadableKVState<EntityNumber, Node>> T nodesState() {
+    public <T extends ReadableKVState<EntityNumber, Node>> T nodesState() {
```

Comment:
State should not be accessed by other classes. Why are we making this public?

Reply:
see below.